src/cryptonote_config.h

```diff
@@ -50,12 +50,17 @@ using namespace std::literals;
 #define CRYPTONOTE_DEFAULT_TX_SPENDABLE_AGE             10
 #define CRYPTONOTE_DEFAULT_TX_MIXIN                     9
 
+#define STAKING_REQUIREMENT_HF16                        15000000000000
 #define STAKING_REQUIREMENT_LOCK_BLOCKS_EXCESS          20
-#define STAKING_PORTIONS                                UINT64_C(0xfffffffffffffffc)
-#define MAX_NUMBER_OF_CONTRIBUTORS                      4
-#define MIN_PORTIONS                                    (STAKING_PORTIONS / MAX_NUMBER_OF_CONTRIBUTORS)
-
-static_assert(STAKING_PORTIONS % 12 == 0, "Use a multiple of twelve, so that it divides evenly by two, three, or four contributors.");
+#define STAKING_PORTIONS_V1                             UINT64_C(0xfffffffffffffffc)
+#define STAKING_PORTIONS_V2                             STAKING_REQUIREMENT_HF16
+#define MAX_NUMBER_OF_CONTRIBUTORS_V1                   4
+#define MAX_NUMBER_OF_CONTRIBUTORS_V2                   10
+#define MIN_PORTIONS_V2                                 (STAKING_PORTIONS_V2 / MAX_NUMBER_OF_CONTRIBUTORS_V2)
+#define MIN_PORTIONS_V1                                 (STAKING_PORTIONS_V1 / MAX_NUMBER_OF_CONTRIBUTORS_V1)
+#define MIN_OPERATOR_PORTION                            (STAKING_PORTIONS_V2 / MAX_NUMBER_OF_CONTRIBUTORS_V1)
```

Comment:
MIN_OPERATOR_PORTION is obviously still needed, but is this MIN_PORTIONS_V2 needed at all?  It seems to me that everyone after the operator is entirely dependent on what was staked before, which makes `MIN_PORTIONS` irrelevant.  (Though there might be some need for the old MIN_PORTIONS_V1 value to stick around for older HF chain handling).

Reply:
is there any reason these are macros besides "tradition"?

Reply:
> is there any reason these are macros besides "tradition"?

Nope.  There's a `config` namespace in here with constexpr values, which is probably a good place to move all of these.

Reply:
that'd probably spiral into a full refactor of everything around it, good idea to omit this task for now.